src/popup/components/ad-hoc-tools-panel.tsx

```diff
@@ -15,7 +15,7 @@ export class AdHocToolsPanel extends React.Component<AdHocToolsPanelProps, {}> {
     private sliceSize: number = 3;
 
     public render(): JSX.Element {
-        const toggles = this.props.diagnosticViewToggleFactory.createTogglesForAdhocToolsPanel();
```

Comment:
The changes look good/make sense to me. I was just curious why this is showing as an addition/removal when the lines in question seem identical?

Reply:
It looks like `createTogglesForAdhocPanel` was renamed, such that the `h` in `Adhoc` was capitalized.

Reply:
yep... I used this opportunity to change from _adhoc_ to _ad**H**oc_